data/cc-agent.target

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Clear Containers Agent Target
+Requires=basic.target
+Requires=cc-agent.service
+Conflicts=rescue.service rescue.target
+After=basic.target rescue.service rescue.target
+AllowIsolate=yes
```

Comment:
ah right, read that sentence before but reading it again now, it make more sense :)